src/System.Text.Json/tests/Serialization/PropertyVisibilityTests.cs

```diff
@@ -67,6 +67,26 @@ public static void PrivateSetter()
             Assert.Null(objCopy.GetMyString());
         }
 
+        [Fact]
+        public static void PrivateSetterPublicGetter()
+        {
+            // https://github.com/dotnet/corefx/issues/37567
```

Comment:
Can we also try to serialize/deserialize and verify round-tripping makes sense in this case?

Reply:
I'm not sure how useful that is in the private case?